core/src/main/java/hudson/model/View.java

```diff
@@ -1045,6 +1045,7 @@ public FormValidation doCheckJobName(@QueryParameter String value) {
      */
     @Restricted(DoNotUse.class)
     public Categories doItemCategories(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException {
+        getOwner().checkPermission(Item.CREATE);
```

Comment:
@daniel-beck Have permission to get this JSON does not involve you need `Item.CREATE`